sample-compose/detekt-config.yml

```diff
@@ -204,7 +204,7 @@ naming:
   TopLevelPropertyNaming:
     active: true
     constantPattern: '[A-Z][_A-Z0-9]*'
-    propertyPattern: '[A-Z][A-Za-z0-9]*'
+    propertyPattern: '[A-Za-z][_A-Za-z0-9]*'
```

Comment:
Saying the `HomeScreen` has a Compose constant style, this rule still reports issue

![image](https://user-images.githubusercontent.com/16315358/206512605-32a2a074-d162-4742-91d0-38d1ffc9f572.png)

![image](https://user-images.githubusercontent.com/16315358/206512650-a073c862-c3b6-483b-9aeb-e2ac8a69d847.png)

My suggestion is

```
    constantPattern: '[A-Z][_A-Za-z0-9]*'
    propertyPattern: '[A-Za-z][A-Za-z\d]*'
```

Reply:
Oh, sorry  @luongvo I think you're right about `constant style.` I just found from [API Guidelines](https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#api-guidelines-for-jetpack-compose)
![Screen Shot 2565-12-09 at 11 48 10](https://user-images.githubusercontent.com/28002315/206626296-4f550a70-80cc-478e-a597-670c07aec643.png)

 I think we should add more part about `Singletons, constants, sealed class, and enum class values` for the convention(`do/don't`) in compass as well. What do you think? cc @hoangnguyen92dn